src/style/index.css

```diff
@@ -31,3 +31,8 @@ ul, li {
   list-style: none;
   padding-inline-start: 0px;
 }
+
+.notification-success {
```

Comment:
For some reason placing this css code in the component file didn't have any affect on the built in styling

Reply:
I think this is great, but we should have a go at creating our own version of these, it'll be quite fun 😄